spec/features/reviews/01_user_adds_a_review_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+feature "user reviews a donut", %{
+  As an authenticated user
+  I would like to add a Review to a Donut
+  So that other people know how my donut experience was
+  } do
+  # ACCEPTANCE CRITERIA
+  # Authenticated user sees donut review form on donut show page
+  # Unauthenticated user does not see review form
```

Comment:
Isn't this just personal preference? I've notice Dan P. uses this in his video lectures and commits with them.